webapp/src/main/webapp/script.js

```diff
@@ -11,3 +11,26 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+/** Fetches notices from the notice container and 
+ * outputs the corresponding functions onto the DOM */
+window.onload=function(){
+  document.forms['myForm'].addEventListener('submit', (event) => {
+    event.preventDefault();
+    fetch('/fileupload', {
+      method: 'POST',
+      body: new FormData(event.target)
+    }).then((resp) => {
+      // Check if output is in JSON format. If not, error has occured.
+      const contentType = resp.headers.get("content-type");
+      if (contentType && contentType.indexOf("application/json") !== -1) {
+        return resp.text().then(data => {
+          this.callCorrespondingFunction(data);
+        });
+      } else {
+        // Upload Error
+        this.alert(resp.text());
```

Comment:
Do we really want a popup on the website?
How about just outputing the text in a red container?

Reply:
Yep ok.

Reply:
Fixed. Error now outputs in the DOM. CSS styling to be added later on to make it a red container.